README.md

```diff
@@ -23,8 +23,15 @@ The top level directory structure groups source code and input files as follow:
 Be sure to download the mkmf submodule prior to beginning.  To use:
 
  1) checkout code via CHECKOUT_code script
+    - ./CHECKOUT_code will checkout necessary files for shields (running with either simple or full coupler)
+    - ./CHECKOUT_code will automatically run ./CHECKOUT_mom6 for mom6/sis2 files
+
  2) cd Build and execute ./COMPILE script with the --help option to see usage
- 3) sample compile:  ./COMPILE shield nh repro 32bit intel
+
+ 3) a) ./COMPILE shield:     will compile shield with simple coupler
+    b) ./COMPILE shieldfull: will compile shield with full coupler (utilizing null modules for ocean, land, ice)
+    c) ./COMPILE shiemom:    will compile mom6, sis2, fv3, gfs as libraries and link them to the full coupler (no null ocean and ice modules.)
+ Example: sample compile:  ./COMPILE shield nh repro 32bit intel
```

Comment:
When I look at this in your branch on Github (https://github.com/JosephMouallem/SHiELD_build/blob/main/README.md)
3 a, b, c and the example blend together.  I'd recommend adding them as  follows:
```
3) compile:
<3 spaces> 1) ./COMPILE shield:     will compile shield with simple coupler
<3 spaces> 2) ./COMPILE shieldfull: will compile shield with full coupler (utilizing null modules for ocean, land, ice)
<3 spaces> 3) ./COMPILE shiemom:    will compile mom6, sis2, fv3, gfs as libraries and link them to the full coupler (no null ocean and ice modules.)
<3 spaces> 3)  Example:  ./COMPILE shield nh repro 32bit intel

Reply:
done